TestFoundation/TestObjCRuntime.swift

```diff
@@ -43,7 +43,7 @@ class TestObjCRuntime: XCTestCase {
         let name = testBundleName()
         XCTAssertEqual(NSStringFromClass(NSObject.self), "NSObject")
         XCTAssertEqual(NSStringFromClass(SwiftClass.self), "\(name).SwiftClass")
-#if DEPLOYMENT_RUNTIME_OBJC || os(Linux)
+#if DEPLOYMENT_RUNTIME_OBJC || os(Linux) || os(Android)
```

Comment:
This should really just switch to the condition we have in place now for importing `SwiftFoundation` vs `Foundation`:

```
#if canImport(SwiftXCTest) && !DEPLOYMENT_RUNTIME_OBJC
… // use SwiftXCTest
#else
… // use XCTest
#endif
```